subsys/debug/openocd.c

```diff
@@ -36,7 +36,9 @@ __attribute__((used, section(".openocd_dbg")))
 size_t _kernel_openocd_offsets[] = {
 	/* Version 0 starts */
 	[OPENOCD_OFFSET_VERSION] = 1,
+#ifndef CONFIG_SMP
```

Comment:
is this going to cause issues w/openocd and the assumed layout of this data?  Wondering if setting the value to 0 is better?

Reply:
0 is probably better, though might break things as well (don't know if openocd/gdb zephyr awareness relies on this).
Openocd Zephyr-SMP support might require more. Maybe instead of working around this, do not allow/use combination of SMP and openocd zephyr awareness for now?

Reply:
`OPENOCD_UNIMPLEMENTED` would be better than zero, but I also like @abrodkin's idea to handle this in KConfig.